src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide(hidden) {
+    this.hidden = true;
+  }
```

Comment:
[FUNCTION NAMING] - The `hide` method should not take any arguments, as per the checklist. It should just set the `hidden` property to `true`. Please remove the `hidden` parameter from the method signature.